portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,9 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function getMessage() {
+  const response = await fetch('/data');
+  const quote = await response.text();
```

Comment:
super nit: maybe call it 'message'? or rename getMessage to getQuote if the responses will always be quotes

Reply:
I think this might eventually be loading comments, but I haven't reached that stage of the walkthrough. I'll change it once I'm sure of what the function is supposed to do!